assets/js/event-providers/woocommerce.js

```diff
@@ -196,4 +196,17 @@
 
 		return mappedItem;
 	}
+
+	/**
+	 * Returns the price of a product formatted with decimal places if necessary.
+	 *
+	 * @since n.e.x.t
+	 *
+	 * @param {string} price                 The price to parse.
+	 * @param {number} [currencyMinorUnit=2] The number decimals to show in the currency.
+	 * @return {number} The price of the product with decimals.
+	 */
+	function formatPrice( price, currencyMinorUnit = 2 ) {
+		return parseInt( price, 10 ) / 10 ** currencyMinorUnit;
```

Comment:
I see this operator so rarely it might be worth explaining exactly what's happening here, because it's not immediately apparent (and I had to read some docs to confirm it was an exponent operator 😅)